src/collection.ts

```diff
@@ -1173,10 +1223,13 @@ export class Collection<TSchema extends Document = Document> {
 
   // Embedded documents overload
   distinct(key: string): Promise<any[]>;
```

Comment:
why did these overloads not make it into api.js?

Reply:
can you clarify which? 
https://github.com/mongodb-js/nodejs-mongodb-legacy/blob/main/mongodb-legacy.d.ts#L619 here?